### DISCUSSION TITLE
Process community rewards cycle: 8/27 -> 9/23

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#250] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 8/27 - 9/23
@Fedias_HS 0.09 damo
@timmyl3g3nd 0.03 damo
AlgoRhythMatic 0.17 damo
BartleDooDerp.algo 0.06 damo
CryptoflipB 0.03 damo
Damo 0.06 damo
DragMZ 0.15 damo
kerrilija 0.15 damo
Maru the Cat 0.03 damo
nearlymad.x 0.03 damo
prince2486 0.03 damo
q2ev 0.15 damo
TOTAL 0.98 damo
```

Please review and comment if there are any questions or concerns. This period uses a 3x coefficient to multiply measurements from the [Rewards Model tab](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=1130085233) from our spreadsheet.

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
Just a note that I just merged two names as I did a mistake in the issues and added a member under wrong name.
